### PR TITLE
Add hook for graphql_query

### PR DESCRIPTION
**Pull request by M0Rf30**
graphql_query needs its templates folder, otherwise the jinja2 loader will fail

### CODE REVIEW

File: src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-graphql_query.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+"""
+PyInstaller hook file for graphql_query. Tested with version 1.0.3.
+"""
+
+from PyInstaller.utils.hooks import collect_data_files
+
+hiddenimports = ["graphql_query"]
```
Comment (rokm) [inline src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-graphql_query.py:17]: In order for the hook to be ran, the module already had to be discovered. So this hidden import is definitely not required.

**Comment by rokm** [comment]
Can you also add a basic test that demonstrates the need for the template files (assuming the basic example from https://github.com/denisart/graphql-query will do?) to [test_libraries.py](https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/master/src/_pyinstaller_hooks_contrib/tests/test_libraries.py) and then add `graphql-query==1.0.3` to [requirements-test-libraries.txt](https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/master/requirements-test-libraries.txt) - this will also trigger the CI on all OSes. 

Then [add a news fragment](https://github.com/pyinstaller/pyinstaller-hooks-contrib/tree/master/news), and we're good to go.

File: src/_pyinstaller_hooks_contrib/tests/test_libraries.py [hunk 2]
```diff
@@ -1214,6 +1214,14 @@ def test_gitlab(pyi_builder):
         import gitlab
         """)
```
Comment (bwoodsend) [inline src/_pyinstaller_hooks_contrib/tests/test_libraries.py:1216]: The linter wants an extra blank line here.

File: src/_pyinstaller_hooks_contrib/tests/test_libraries.py [hunk 3]
```diff
@@ -1214,6 +1214,14 @@ def test_gitlab(pyi_builder):
...
 
+@importorskip('graphql_query')
+def test_graphql_query(pyi_builder):
+    pyi_builder.test_source("""
+        from graphql_query import Operation, Query
+        hero = Query(name="hero", fields=["name"])
+        operation = Operation(type="query", queries=[hero])
+        print(operation.render())
+        """)
```
Comment (bwoodsend) [inline src/_pyinstaller_hooks_contrib/tests/test_libraries.py:1225]: ... and here.

**Comment by bwoodsend** [comment]
That's it released in `pyinstaller-hooks-contrib==2023.3`.

**Pull request merged**